pkg/cpio/newc.go

```diff
@@ -183,19 +185,82 @@ type reader struct {
 	pos int64
 }
 
+type discarder struct {
```

Comment:
It wasn't obvious to me that a discarder exists to provide an implementation of ReadAt where Seek isn't supported.  Might be worth a comment.

Reply:
done.